src/api/Feed.js

```diff
@@ -12,7 +12,7 @@ import VersionsAPI from './Versions';
 import TasksAPI from './Tasks';
 import TaskAssignmentsAPI from './TaskAssignments';
 import { getBadItemError, getBadUserError, isUserCorrectableError } from '../util/error';
-import messages from '../components/messages';
+import messages from '../elements/common/messages';
```

Comment:
It would be great to alias both `messages` and `constants` globally at some point.

Reply:
Yeah